### PR TITLE
Reintroduce governance contracts

### PR DESCRIPTION
**Pull request by goran-ethernal**
This PR reintroduces governance contracts.

### CODE REVIEW

File: contracts/blade/governance/ChildGovernor.sol [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    ) public initializer {
+        __Governor_init("ChildGovernor");
+        __GovernorTimelockControl_init(timelock_);
+        __GovernorVotes_init(token_);
+        __GovernorVotesQuorumFraction_init(quorumNumerator_);
+
+        _networkParams = NetworkParams(networkParams);
+    }
+
+    // TODO: adjust values for block time of child chain
```
Comment (Stefan-Ethernal) [inline contracts/blade/governance/ChildGovernor.sol:35]: Remove?

File: contracts/blade/governance/ChildGovernor.sol [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    function votingDelay() public view override returns (uint256) {
+        return _networkParams.votingDelay();
+    }
+
+    function votingPeriod() public view override returns (uint256) {
+        return _networkParams.votingPeriod();
+    }
+
+    // END TODO
```
Comment (Stefan-Ethernal) [inline contracts/blade/governance/ChildGovernor.sol:45]: Remove?

**Review by Stefan-Ethernal — APPROVED** [review]

File: contracts/blade/governance/ChildGovernor.sol (hunk 1 above)
Comment (goran-ethernal) [reply contracts/blade/governance/ChildGovernor.sol]: Fix: https://github.com/Ethernal-Tech/blade-contracts/pull/6/commits/b8ea920717ade73fd84fb562a19a88e8cfc3bc25

File: contracts/blade/governance/ChildGovernor.sol (hunk 2 above)
Comment (goran-ethernal) [reply contracts/blade/governance/ChildGovernor.sol]: Fix: https://github.com/Ethernal-Tech/blade-contracts/pull/6/commits/b8ea920717ade73fd84fb562a19a88e8cfc3bc25

**Pull request merged**